src/nvim/options.lua

```diff
@@ -539,7 +539,7 @@ return {
       full_name='cursorline', abbreviation='cul',
       type='bool', scope={'window'},
       vi_def=true,
-      redraw={'current_window'},
+      redraw={'current_window_only'},
```

Comment:
That value doesn't exist, see https://github.com/neovim/neovim/blob/master/src/nvim/options.lua#L20-L21